app/src/components/basic-search/basic-search-helper.js

```diff
@@ -1,10 +1,10 @@
-import { geoJSONToGeometryJSON } from 'geospatialdraw/bin/geometry/utilities'
-import { geoToFilter } from '../location'
-import { makeSearchGeoIdForFilter } from '../query-builder/filter'
+// import { geoJSONToGeometryJSON } from "geospatialdraw/bin/geometry/utilities";
```

Comment:
Uncomment or remove

Reply:
ah good catch, i forgot to add the location stuff back to basic search